apps/openmw/mwmechanics/aisequence.cpp

```diff
@@ -28,6 +28,12 @@ void AiSequence::copy (const AiSequence& sequence)
     for (std::list<AiPackage *>::const_iterator iter (sequence.mPackages.begin());
         iter!=sequence.mPackages.end(); ++iter)
         mPackages.push_back ((*iter)->clone());
+
+    // We need to keep an AiWander storage, if present - it has a state machine.
+    // Not sure about another temporary storages
+    AiWanderStorage* copy = sequence.mAiState.copy<AiWanderStorage>();
+    if (copy != nullptr)
+        mAiState.store<AiWanderStorage>(*copy);
```

Comment:
`store` makes another copy of `copy` object and `copy` pointer memory is leaked here.